### ISSUE TITLE
Implement lane assignment for the bowling area

### ISSUE THREAD

**Issue by trundler-dev** [trundler-dev/sbox-sports#50]
Multiple instances of bowling will share the same alley in-game. We should have a system that assigns lanes. This should be easy, but this will ensure our edge cases are covered.

- Assign one lane to each instance
- If an instance is disbanded (all players in the instance leave), the lane is freed up
- If all lanes are full, a bowling instance cannot be created (go to another server, or figure out a better solution?)
- Ensure lanes are properly reset
- Ensure players are spawned in the correct area, and not blocking other players' cameras.

**Comment by trundler-dev** [comment]
This is blocked by #4

**Comment by DevulTj** [comment]
Ideally we should just set these up as different gamemode instances

**Comment by DoctorGurke** [comment]
yeah each lane should be a gamemode instance with up to 6 players
